src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -0,0 +1,120 @@
+        c.argument('tags', arg_type=tags_type)
+        c.argument('sku', options_list=['--sku'], arg_type=get_enum_type(SkuName))
+        c.argument('capacity', type=int, help='Capacity for Sku')
+        c.argument('is_auto_inflate_enabled', options_list=['--enable-auto-inflate'], arg_type=get_three_state_flag(), help='A boolean value that indicates whether AutoInflate is enabled for eventhub namespace.')
+        c.argument('maximum_throughput_units', type=int, help='Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units. ( 0 if AutoInflateEnabled = true)')
+
+    # region Namespace Authorizationrule
+    with self.argument_context('eventhubs namespace authorization-rule list') as c:
+        c.argument('namespace_name', options_list=['--namespace-name'], help='name of the Namespace')
+
+    for scope in ['eventhubs namespace authorization-rule', 'eventhubs namespace authorization-rule keys list', 'eventhubs namespace authorization-rule keys renew']:
+        with self.argument_context(scope) as c:
+            c.argument('authorization_rule_name', arg_type=name_type, id_part='child_name_1', help='name of the Namespace AuthorizationRule')
+            c.argument('namespace_name', options_list=['--namespace-name'], id_part='name', help='name of the Namespace')
+
+    for scope in ['eventhubs namespace authorization-rule create', 'eventhubs namespace authorization-rule update', 'eventhubs eventhub authorization-rule create', 'eventhubs eventhub authorization-rule update']:
+        with self.argument_context(scope) as c:
+            c.argument('rights', arg_type=rights_arg_type)
+
+    with self.argument_context('eventhubs namespace authorization-rule keys renew') as c:
+        c.argument('key_type', arg_type=key_arg_type)
+
+
+# region - Eventhub Create
+    with self.argument_context('eventhubs eventhub') as c:
+        c.argument('event_hub_name', arg_type=name_type, id_part='child_name_1', completer=get_eventhubs_command_completion_list, help='Name of Eventhub')
+
+    for scope in ['eventhubs eventhub create', 'eventhubs eventhub update']:
+        with self.argument_context(scope) as c:
+            c.argument('message_retention_in_days', options_list=['--message-retention'], type=int, help='Number of days to retain events for this Event Hub, value should be 1 to 7 days')
+            c.argument('partition_count', type=int, help='Number of partitions created for the Event Hub, allowed values are from 1 to 32 partitions.')
+            c.argument('status', arg_type=get_enum_type(['Active', 'Disabled', 'SendDisabled', 'ReceiveDisabled']), help='Status of Eventhub')
+            c.argument('enabled', options_list=['--enable-capture'], arg_type=get_three_state_flag(), help='A boolean  value that indicates whether capture description is enabled.')
```

Comment:
nit: There are two spaces between `boolean  value`. There should only be one.

Reply:
resolved